modules/virt-about-hot-plugging-virtual-disks.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-hot-plugging-virtual-disks_{context}"]
+= About hot-plugging virtual disks
+
+When you _hot-plug_ a virtual disk, you attach a virtual disk to a virtual machine instance while the virtual machine is running. 
+
+When you _hot-unplug_ a virtual disk, you detach a virtual disk from a virtual machine instance while the virtual machine is running.
+
+Only data volumes and persistent volume claims (PVCs) can be hot-plugged and hot-unplugged. You cannot hot-plug or hot-unplug container disks.
+
+After you hot-plug a virtual disk, it remains attached until you detach (unplug) it, even if you restart the virtual machine.
```

Comment:
After you hot-plug a virtual disk, it remains attached until you detach (unplug) it, even if you restart the virtual machine. -> After you hot-plug a virtual disk, it remains attached even after restarting the virtual machine, until you detach (unplug) it.